.buildkite/pipeline.yml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - label: Build and test
+    command: |
+      nix-build -A capability --no-out-link
```

Comment:
I've verified this runs the tests as well and on test failure the error message is printed to console. So, CI should also display test errors.

Reply:
Yes, that's the idea.